### PR TITLE
Upgrade to new flow

### PR DESCRIPTION
**Pull request by DanielHeath**

### CODE REVIEW

File: .flowconfig [hunk 1]
```diff
@@ -16,6 +16,8 @@ module.system=node
 suppress_type=$FlowIssue
 suppress_comment= \\(.\\|\n\\)*\\flow:disable
 strip_root=false
+esproposal.class_static_fields=warn
```
Comment (danni) [inline .flowconfig]: What do these?

Comment (DanielHeath) [inline .flowconfig]: We should be able to get rid of many of the flow:disable comments. Also valid is `=ignore`.

Comment (danni) [inline .flowconfig]: Perhaps it should be ignore?

**Comment by danni** [comment]
++

**Pull request merged**